receiptui/src/main/java/com/hyperwallet/android/ui/receipt/HyperwalletReceiptUi.java

```diff
@@ -52,11 +52,11 @@ public static synchronized HyperwalletReceiptUi getInstance(
      * @param lockScreenToPortrait if set <code>true</code> screen will be locked to Portrait mode;
      *                             otherwise <code>false</code> screen will follow whatever the
      *                             device orientation is directed.
-     * @return an Intent with the data necessary to launch the {@link ListUserReceiptActivity}
+     * @return an Intent with the data necessary to launch the {@link TabbedListReceiptActivity}
      */
     public Intent getIntentListUserReceiptActivity(@NonNull final Context context, final boolean lockScreenToPortrait) {
-        Intent intent = new Intent(context, ListUserReceiptActivity.class);
-        intent.putExtra(ListUserReceiptActivity.EXTRA_LOCK_SCREEN_ORIENTATION_TO_PORTRAIT, lockScreenToPortrait);
+        Intent intent = new Intent(context, TabbedListReceiptActivity.class);
```

Comment:
Why are you updating the existing method for user receipts? Please refer to the design diagram, we are creating a new method for tabbed view..

Reply:
Sorry. Updated with new method.